build.sbt

```diff
@@ -438,44 +510,45 @@ lazy val tomcatServer = libraryProject("tomcat-server")
   .dependsOn(servlet % "compile;test->test")
 
 // `dsl` name conflicts with modern SBT
-lazy val theDsl = libraryProject("dsl")
+lazy val theDsl = libraryCrossProject("dsl", CrossType.Pure)
   .settings(
     description := "Simple DSL for writing http4s services",
     startYear := Some(2013),
   )
   .dependsOn(core, testing % "test->test")
 
-lazy val jawn = libraryProject("jawn")
+lazy val jawn = libraryCrossProject("jawn", CrossType.Pure)
   .settings(
     description := "Base library to parse JSON to various ASTs for http4s",
     startYear := Some(2014),
     libraryDependencies ++= Seq(
-      jawnFs2,
-      jawnParser,
+      jawnFs2.value,
+      jawnParser.value,
     )
   )
   .dependsOn(core, testing % "test->test")
 
-lazy val boopickle = libraryProject("boopickle")
+lazy val boopickle = libraryCrossProject("boopickle", CrossType.Pure)
   .settings(
     description := "Provides Boopickle codecs for http4s",
     startYear := Some(2018),
     libraryDependencies ++= Seq(
-      Http4sPlugin.boopickle
+      Http4sPlugin.boopickle.value
     ),
   )
   .dependsOn(core, testing % "test->test")
 
-lazy val circe = libraryProject("circe")
+lazy val circe = libraryCrossProject("circe", CrossType.Pure)
   .settings(
     description := "Provides Circe codecs for http4s",
     startYear := Some(2015),
     libraryDependencies ++= Seq(
-      circeCore,
-      circeJawn,
-      circeTesting % Test
+      circeCore.value,
+      circeTesting.value % Test
     )
   )
+  .jvmSettings(libraryDependencies += circeJawn.value)
+  .jsSettings(libraryDependencies += circeJawn15.value)
```

Comment:
`circe-jawn` only cross-builds for JS as of 0.15.0-M1. JVM continues on 0.14.x series.